Source/Launcher/General/Direct3D.cs

```diff
@@ -10,10 +10,12 @@
 using System.Runtime.CompilerServices;
 using System.Windows.Forms;
 using SharpDX.Direct3D9;
+using CodeImp.Bloodmasters.Launcher.Interface;
 
-namespace CodeImp.Bloodmasters.Launcher;
 
-internal sealed class Direct3D
+namespace CodeImp.Bloodmasters.Launcher.General;
+
+internal sealed class CustomDirect3D : Direct3D
```

Comment:
What happens here? Why inherit `Direct3D`?

Reply:
Because it gives an ambiguous error reference between 'CodeImp.Bloodmasters.Launcher.General.Direct3D' and 'SharpDX.Direct3D9.Direct3D'	Bloodmasters.Launcher

Reply:
Better use `using Direct3D = CodeImp.Bloodmasters.Launcher.General.Direct3D` or some other solution instead of this dubious inheritance.

Reply:
Renamed as it was